next/components/thumbprint-components/props-table/component-definition.ts

```diff
@@ -0,0 +1,24 @@
+export interface ComponentDefinition {
+    displayName: string;
+    description: string;
+    methods: unknown[];
```

Comment:
Is there a reason this is unknown?

Reply:
This only ever gets populated by class components, I think. I've updated this PR to expand on this type definition a bit.